ui/lib/apps/ClusterInfo/components/InstanceTable.tsx

```diff
@@ -78,6 +79,19 @@ export default function ListPage() {
   } = useClientRequest((cancelToken) =>
     client.getInstance().getPDTopology({ cancelToken })
   )
+  const errMessages = useMemo(() => {
+    let errMsgs: string[] = []
+    if (errTiDB) {
+      errMsgs.push('Load TiDB instances failed')
```

Comment:
We'd better provide detailed error information

Reply:
I think so.